SteamPrefill/SteamManager.cs

```diff
@@ -346,5 +348,78 @@ private async Task<BenchmarkWorkload> BuildBenchmarkWorkloadAsync(List<uint> app
         }
 
         #endregion
+
+        #region Status
+
+        public async Task CurrentlyDownloadedAsync(SortOrder sortOrder, string sortColumn)
+        {
+            await _cdnPool.PopulateAvailableServersAsync();
+
+            List<uint> appIds = LoadPreviouslySelectedApps();
+            ByteSize totalSize = new ByteSize();
+            Dictionary<string, ByteSize> index = new Dictionary<string, ByteSize>();
+
```

Comment:
Throw in `await _appInfoHandler.RetrieveAppMetadataAsync(appIds);` right around here.  This will make the whole thing run significantly faster by bulk loading the app metadata at once, rather than one by one as it currently is doing.